### PR TITLE
Strike, Stock Strike, Get Up

### PR DESCRIPTION
**Pull request by Schizosaccaromyces**
Added flyovers for those three skills.

### CODE REVIEW

File: Ayylmaolwotc/Config/XComGameData_AyyLmao.ini [hunk 1]
```diff
@@ -73,7 +73,10 @@
 +FLYOVERS_ON_SHOOTERS=CyclicFire
 +FLYOVERS_ON_SHOOTERS=CyclicFire2
 +FLYOVERS_ON_SHOOTERS=CyclicFire3
-+FLYOVERS_ON_SHOOTERS=Demolition
```
Comment (Grobobobo) [inline Ayylmaolwotc/Config/XComGameData_AyyLmao.ini:76]: Why Did you remove demolition  here?

**Review by Grobobobo — CHANGES_REQUESTED** [review]

File: Ayylmaolwotc/Config/XComGameData_AyyLmao.ini (hunk 1 above)
Comment (Schizosaccaromyces) [reply Ayylmaolwotc/Config/XComGameData_AyyLmao.ini:76]: Stupidity on my part. Was accidental.
